### ISSUE TITLE
Home page for M2 APIs

### ISSUE THREAD

**Issue by GozdeUnver** [bounswe/2021SpringGroup5#130]
I will create a home page for our M2 project where buttons to go different API functionalities will be located.

**Comment by GozdeUnver** [comment]
I created a home app that links to other apps and has a home html file that has buttons to those other apps. I connected my weatherCondition app to home app and created button on the html file for weatherCondition app. I pushed it to my branch called -gozdebranch-

**Comment by GozdeUnver** [comment]
@cagin24 will add the remaining url connections to the home app after other team members push their codes to github and he will also add buttons to the html code for those apps.

**Comment by GozdeUnver** [comment]
We decided with @cagin24 that I will make the modifications and he will review them before merging to the master. First pull request is [here](https://github.com/bounswe/2021SpringGroup5/pull/168)

**Comment by GozdeUnver** [comment]
I added a button for musicapi and added its path to the home/urls.py. musicapi was not correctly added as an app in the settings.py file so the program was not working so I corrected it. You can see the pull request [here](https://github.com/bounswe/2021SpringGroup5/pull/182)

**Comment by GozdeUnver** [comment]
I added buttons and paths of exchangeRaeAPI, airquality, core and randomActivity. I modified settings.py and added these apps correctly. When I tried to run the project, nbastats, sport_relation and core apps were giving errors so I contacted with @atasoyfurkan how to handle these errors. His app sport_relation and @salih997 's app nbastats were conflicting on their templates because their directory design for templates were not appropriate. I pushed the modifications I did to the home branch and added a [pull request](https://github.com/bounswe/2021SpringGroup5/pull/190) and added @atasoyfurkan as the reviewer. He will look at the mistakes as well.

**Comment by GozdeUnver** [comment]
@atasoyfurkan and I reviewd the final code. We decided on making some modifications. Detailed information is in this [pull request](https://github.com/bounswe/2021SpringGroup5/pull/207). Home page and the development of the project is done

**Comment by atasoyfurkan** [comment]
I reviewed the final code, and it has some bugs like Gozde mentioned. I solved them like that: 
* 5 app did not have correct template order. It has to be /templates/app_name/some.html. I change to correct by this way, bugs are resolved.
* Core app did not have root '/' view I change /authenctication to root '/' so problem is resolved.
* Core app did have name spacing issue. That was causing to error in the main admin app so I add name spacing to Core app.
* requirements.txt had wrong requirement "decouple" I change it with correct one "python-decouple".
* I change token in settings.py for musicapi (with help of Gozde and Kemal). Previous one was causing to problem.
